content/docs/csidriver/installation/helm/isilon.md

```diff
@@ -111,6 +111,7 @@ kubectl create -f deploy/kubernetes/snapshot-controller
    | ***PLATFORM ATTRIBUTES*** | | | |   
    | endpointPort | Define the HTTPs port number of the PowerScale OneFS API server. If authorization is enabled, endpointPort should be the HTTPS localhost port that the authorization sidecar will listen on. This value acts as a default value for endpointPort, if not specified for a cluster config in secret. | No | 8080 |
    | skipCertificateValidation | Specify whether the PowerScale OneFS API server's certificate chain and hostname must be verified. This value acts as a default value for skipCertificateValidation, if not specified for a cluster config in secret. | No | true |
+   | isiAuthType | Indicates whether the authentication will be session-based or basic. If set to 1 then it follows as session-based authentication else basic authentication | No | 0 |
```

Comment:
Can be rephrased as 
Indicates the authentication method to be used. If set to 1 then it follows as session-based authentication else basic authentication